packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/resultTemplates/defaultResultTemplate.html

```diff
@@ -1,20 +1,38 @@
 <template>
-    <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}>
-      <c-quantic-result-label slot="label" result={result}></c-quantic-result-label>
-      <div slot="badges" class="slds-grid slds-grid_vertical-align-center slds-m-vertical_xx-small slds-m-right_small">
-        <div class="slds-m-right_xx-small">
-          <c-quantic-result-badge variant="recommended" result={result}></c-quantic-result-badge>
-        </div>
-        <c-quantic-result-badge variant="featured" result={result}></c-quantic-result-badge>
+  <c-quantic-result-template
+    is-any-preview-open={isAnyPreviewOpen}
+    result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}
+    hide-label
+  >
+    <div
+      slot="badges"
+      class="slds-grid slds-grid_vertical-align-center slds-m-vertical_xx-small slds-m-right_small"
+    >
+      <c-quantic-colored-result-badge
+        label={result.raw.filetype}
```

Comment:
not too sure what the label value should be here, I put `result.raw.filetype` for now, because if I put `result.raw.objecttype` the results in issues and files tabs it causes render error. What do you think we should put as labels? Since its an example, do we simply hardcode it?

Reply:
@MehdiMitiche